pkg/operator/ceph/csi/template/rbd/csi-rbdplugin.yaml

```diff
@@ -67,7 +67,7 @@ spec:
             - "--metricsport={{ .RBDGRPCMetricsPort }}"
             - "--metricspath=/metrics"
             - "--enablegrpcmetrics={{ .EnableCSIGRPCMetrics }}"
-            - "--stagingpath={{ .KubeletDirPath }}/plugins/kubernetes.io/csi/pv/"
+            - "--stagingpath={{ .KubeletDirPath }}/plugins/kubernetes.io/csi/"
```

Comment:
Does this change only apply to K8s 1.24 or newer? Or is it also ok to change on older versions as well? If it's only for 1.24+, we will need a version check for different behavior.

Reply:
This change is needed for K8s >= 1.24, but we have prepared CSI-driver with https://github.com/ceph/ceph-csi/pull/3207, so that it will work with K8s < 1.24 as well.

The corner case here would be:
If someone uses an older CSI driver which doesn't have https://github.com/ceph/ceph-csi/pull/3207 and uses a newer rook version that has this PR in combination with K8s < 1.24, then they might see NodeStageVolume() failure. Not sure if we should deal with this combination though.

@Madhu-1 @travisn WDYT?

Reply:
IMO this is required only for nbd recovery (nbd support is still in alpha state), lets's get the cephcsi release next week and merged this one and backport it to 19.x so that we don't need to worry about anything as cephcsi new version will handle it.

If someone is using an older version of cephcsi/Rook with kubernetes 1.24 and seeing this issue we can ask them to upgrade to the latest version to fix it.

Reply:
Will the csi version with this change (v3.6.3?) be required now? In other words, will we need to set the min version of the csi driver to v3.6.3 instead of allowing older csi versions? Almost everyone likely uses the default version that comes with Rook, but I don't know how many users would be affected if we increase the min csi version.

Reply:
Yes, we can do it, As I mentioned earlier only PVC with nbd driver is affected by this. Not sure anyone is using it in production as the support is still Alpha,  we can change the default to v3.6.3 or ask them to update the CSI version to 3.6.3 when someone reports the issue.

Reply:
Ok, so no need to update the min csi version, thanks